python/pyspark/logger/tests/test_logger.py

```diff
@@ -0,0 +1,77 @@
+# -*- encoding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+import json
+from io import StringIO
+from pyspark.logger.logger import PySparkLogger
+
+
+class LoggerTestsMixin:
+    def setUp(self):
+        self.logger = PySparkLogger.get_logger("TestLogger", stream=StringIO())
```

Comment:
why not configure the logger so that you keep the stream as a member variable that you can access instead of punching through the private interface of the logger?

This saves you from making `handler` a public member and the whole logger behaves much more standard.

Reply:
Makes sense to me. Will address the comment.

Thanks for the suggestion!

Reply:
Updated! Now this behavior the same as standard `logging` module :-)